streampipes-sdk/src/main/java/org/apache/streampipes/sdk/builder/AbstractConfigurablePipelineElementBuilder.java

```diff
@@ -1013,7 +1013,13 @@ public K requiredMultiValueSelectionFromContainer(Label label,
 
   public K requiredRuntimeResolvableTreeInput(Label label,
```

Comment:
Do we still need this method? You can achieve the same by using  `requiredRuntimeResolvableTreeInput(Label label,List<String> dependsOn, boolean resolveDynamically)`

Reply:
This is meant to be a convenience method and also ensures backwards compatibility since we've introduced the `resolveDynamically` boolean flag.